src/main/java/frc/robot/subsystems/apriltagvision/AprilTagVisionIOLimelight.java

```diff
@@ -0,0 +1,130 @@
+package frc.robot.subsystems.apriltagvision;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.Alert;
+import edu.wpi.first.wpilibj.Alert.AlertType;
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.subsystems.apriltagvision.AprilTagVisionConstants.*;
+import frc.robot.util.LimelightHelpers;
+import frc.robot.util.PoseManager;
+
+public class AprilTagVisionIOLimelight implements AprilTagVisionIO {
+  private String name;
+  private double[] position;
+
+  private static final double disconnectedTimeout = 0.5;
+  private final Alert disconnectedAlert;
+  private double lastTimestamp = 0;
+
+  private final double DEFAUlT_CROP = 0.9;
+  private final double CROP_BUFFER = 0.1;
+
+  public AprilTagVisionIOLimelight(String camName) {
+    name = camName;
+    double[] position;
+    switch (name) {
+      case AprilTagVisionConstants.reefName:
```

Comment:
take advantage of static import like this:
```suggestion
      case reefName:
```